### ISSUE TITLE
missing values problem

### ISSUE THREAD

**Issue by mdsumner** [hypertidy/lazyraster#8]
Ok when no missing values: 

```R

s <- vapour::vapour_sds_names(raadfiles::altimetry_daily_files()$fullname[1])$subdataset[4]


plot(lazycrop(lazyraster(s), extent(100, 120, -50, -40)))
Warning messages:
1: In raster_info_cpp(filename = datasourcename, min_max = min_max) :
  GDAL Message 1: The dataset has several variables that could be identified as vector fields, but not all share the same primary dimension. Consequently they will be ignored.
2: In raster_info_cpp(filename = datasourcename, min_max = min_max) :
  GDAL Message 1: The dataset has several variables that could be identified as vector fields, but not all share the same primary dimension. Consequently they will be ignored.
```

Not ok when we traverse missing pixels (no plot)

```R
plot(lazycrop(lazyraster(s), extent(100, 220, -50, -40)))
Warning messages:
1: In raster_info_cpp(filename = datasourcename, min_max = min_max) :
  GDAL Message 1: The dataset has several variables that could be identified as vector fields, but not all share the same primary dimension. Consequently they will be ignored.
2: In raster_info_cpp(filename = datasourcename, min_max = min_max) :
  GDAL Message 1: The dataset has several variables that could be identified as vector fields, but not all share the same primary dimension. Consequently they will be ignored.
3: In x - r[1] : NAs produced by integer overflow
4: In r[2] - r[1] : NAs produced by integer overflow
```

**Comment by mdsumner** [comment]
Fixed in 38c2e58

We'll see if that's a good idea or not

**Comment by mdsumner** [comment]
Probably better if we get the missing value itself on vapour_raster_info
